packages/components/src/ConditionalFilter/GroupFilter.js

```diff
@@ -267,21 +309,37 @@ class Group extends Component {
                         groups.map(({
                             value: groupValue,
                             onSelect,
+                            groupSelectable,
                             label: groupLabel,
                             id: groupId,
                             type,
                             items,
                             ...group
                         }, groupKey) => {
-                            const filteredItems = this.mapItems({ groupValue, onSelect, groupLabel, groupId, type, items, ...group }, groupKey)
+                            const filteredItems = this.mapItems({ groupValue, onSelect, groupLabel, groupId, groupSelectable, type, items, ...group }, groupKey)
                             .filter(Boolean);
-                            return (<SelectGroup
-                                {...group}
-                                key={groupId || groupValue || groupKey}
-                                value={groupId || groupValue || groupKey}
-                                label={groupLabel || ''}
-                                id={groupId || `group-${groupValue || groupKey}`}
-                            >{filteredItems}</SelectGroup>);
+
+                            return (
+                                <div key={groupId || groupValue || groupKey}>
```

Comment:
Let's hope that this won't break the UI (PF select has ways of breaking stuff). With the new menu component we should sink in some time and switch this component to the new approach. That way we won't have all these problems.